### ISSUE TITLE
[Accessibility] DatePicker control should highlight today in high contrast mode

### ISSUE THREAD

**Issue by daweffe** [OfficeDev/office-ui-fabric-react#2656]
### Bug Report

- __Package version(s)__: 4.21.3
- __Browser and OS versions__: IE/Edge

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (No)

Requested priority: (Normal) 

Products/sites affected: Planner

#### Describe the issue:

Today's date in the date picker is not highlighted when in high contrast mode in IE or Edge.  

I tried fixing this locally with CSS, but uncovered a bug in Edge.  As seen in the images below, Edge is overriding the background around the text by setting it to the general background color.  In High Contrast #1, this looks bad, but the number is legible.  In HC Black and HC White, it means the text and background are the same color.
![edge high contrast bug - high contrast 1](https://user-images.githubusercontent.com/29383146/29797194-a0b204a6-8c0a-11e7-8033-f30ff5e563b5.PNG)
![edge high contrast bug - high contrast black](https://user-images.githubusercontent.com/29383146/29797203-a814bff4-8c0a-11e7-9b2f-f7b649e07c94.PNG)
![edge high contrast bug - high contrast white](https://user-images.githubusercontent.com/29383146/29797205-a9ceafc6-8c0a-11e7-93a1-d383320c2154.PNG)

#### Actual behavior:

Today has the regular background and text colors.

#### Expected behavior:

Today should use the selected text background and foreground colors, see images above.

Codepen (enable high contrast mode in windows and use Edge):
https://codepen.io/anon/pen/ZJqqMq

**Comment by daweffe** [comment]
The Edge team should be consulted and advised of their bug.  Has anyone done this yet?

**Comment by daweffe** [comment]
To make it work in Edge you have to add:

-ms-high-contrast-adjust: none;

**Comment by daweffe** [comment]
To make it work in Edge you have to add:

-ms-high-contrast-adjust: none;
